src/specs/01_basic/revert_reason.spec

```diff
@@ -1,6 +1,9 @@
 # Revert Reason is present on Transaction Receipts
 
- Tags: revert-reason, networks/typical::raft, networks/typical::istanbul, networks/typical-besu::ibft2
+ Tags: revert-reason,
```

Comment:
non-blocking: maybe we should simply add it to the `basic` tag and remove the networks (it should capture all networks that are typical) as long as all of those have the argument to geth. wdyt?

Reply:
i think the reason why tags were added for each `typical` network was because --revertreason has only been added to `typical` nodes, correct me if I'm wrong.  Adding the `basic` tag would run the tests on the other non-revertreason enabled networks too.

Reply:
yes, correct. but what I mean here is that we can add to all typical networks as those are the ones that run basic I think. this is non-blocking so I can check it later.

Reply:
plugin networks also run basic tagged tests - obviously revertreason can be enabled on those networks too but if so then i think it makes sense to do that as a separate change

Reply:
yes, agreed. closing